src/glue/resources/table_columns.yaml

```diff
@@ -0,0 +1,461 @@
+tables:
+  EnrolledParticipants:
+    columns:
+      - Name: ParticipantIdentifier
+        Type: string
+      - Name: GlobalKey
+        Type: string
+      - Name: EmailAddress
+        Type: string
+      - Name: FirstName
+        Type: string
+      - Name: MiddleName
+        Type: string
+      - Name: LastName
+        Type: string
+      - Name: Gender
+        Type: string
+      - Name: DateOfBirth
+        Type: string
+      - Name: SecondaryIdentifier
+        Type: string
+      - Name: PostalCode
+        Type: string
+      - Name: EnrollmentDate
+        Type: string
+      - Name: CustomFields
+        Type: struct<AppleHealthEnabled:string,AppleHealthRecordsEnabled:string,GoogleFitEnabled:string,Reminder1Enabled:string,Reminder2Enabled:string,ReminderTime1:string,ReminderTime2:string,SkipConsent:string,Symptoms:string,Treatments:string>
+      - Name: EventDates
+        Type: string
+      - Name: UtcOffset
+        Type: string
+      - Name: TimeZone
+        Type: string
+      - Name: PreferredLanguage
+        Type: string
+  HealthKitV2Statistics:
+    columns:
+      - Name: HealthKitStatisticKey
+        Type: string
+      - Name: ParticipantIdentifier
+        Type: string
+      - Name: StartDate
+        Type: string
+      - Name: Date
+        Type: string
+      - Name: Value
+        Type: string
+      - Name: Units
+        Type: string
+      - Name: InsertedDate
+        Type: string
+      - Name: export_start_date
+        Type: boolean
+      - Name: export_end_date
+        Type: boolean
+  HealthKitV2Samples:
+    columns:
+      - Name: HealthKitSampleKey
+        Type: string
+      - Name: ParticipantIdentifier
+        Type: string
+      - Name: Type
+        Type: string
+      - Name: StartDate
+        Type: string
+      - Name: Date
+        Type: string
+      - Name: Value
+        Type: string
+      - Name: Units
+        Type: string
+      - Name: Source
+        Type: struct<Identifier:string,Name:string,Version:string,OperatingSystemVersion:string,ProductType:string>
+      - Name: Device
+        Type: struct<Name:string,Model:string,Manufacturer:string,HardwareVersion:string,SoftwareVersion:string,FirmwareVersion:string,LocalIdentifier:string,FDAIdentifier:string>
+      - Name: Metadata
+        Type: struct<HKMetadataKeySyncVersion:string,HKVO2MaxTestType:string,HKMetadataKeySyncIdentifier:string>
```

Comment:
For these arbitrary JSON fields, reference the tables in `test-database` that I created using the crawlers. You'll find a few more fields there. For example, this is the struct for the Metadata field:

```
{
  "metadata": {
    "HKWasUserEntered": "string",
    "HKAlgorithmVersion": "string",
    "HKMetadataKeyHeartRateMotionContext": "string",
    "HKDateOfEarliestDataUsedForEstimate": "string",
    "HKMetadataKeyAppleDeviceCalibrated": "string",
    "HKTimeZone": "string",
    "HKMetadataKeySyncVersion": "string",
    "HKVO2MaxTestType": "string",
    "HKMetadataKeySyncIdentifier": "string",
    "HKMetadataKeyDevicePlacementSide": "string"
  }
}
```